setup.py

```diff
@@ -31,6 +33,99 @@
         version = match.group(1)
 
 
+def _get_help_output(topic, lines=False):
+    TRANSLATE = {
+        'tar-content': 'tar_content',
+    }
+    if topic in TRANSLATE:
+        topic = TRANSLATE[topic]
+    return _run_snapcraft(['help', topic], lines)
+
+
+def _run_snapcraft(cmd, lines=False):
+    cmd = ['./bin/snapcraft']+cmd
+    command = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    output = command.communicate()[0]
+
+    if sys.version_info.major == 3:
+        output = output.decode('utf-8')  # bytes to str
```

Comment:
I think I would prefer just to ignore python2. @sergiusens do you know if there's a good reason to keep this check?